src/Sulu/Component/Rest/ListBuilder/ListRestHelper.php

```diff
@@ -76,7 +76,12 @@ public function getSortOrder()
      */
     public function getLimit()
     {
-        return $this->getRequest()->get('limit', 10);
+        $default = 10;
+        if ($this->getRequest()->getRequestFormat() === 'csv') {
+            $default = null;
+        }
+
+        return $this->getRequest()->get('limit', $default);
```

Comment:
@wachterjohannes Where is tested if that really works? Are there any test cases with more than 10 entities? Otherwise I would add a unit test here.